src/main/scala/org/globalnames/parser/Node.scala

```diff
@@ -1,8 +1,86 @@
 package org.globalnames.parser
+
 import scala.collection._
+import org.json4s._
+import org.json4s.JsonDSL._
+import org.json4s.jackson.JsonMethods._
 
 case class Node(
   normalized: String = "",
   canonical: String = "",
-  hybrid: Boolean = false
+  hybrid: Boolean = false,
+  parser_run: Int = 1,
+  details: Option[Vector[Name]] = None,
```

Comment:
Why `Option[Vector]` required? Why can't be just `Vector.empty` instead of `None`?